maven-plugin/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-schema-registry-parent</artifactId>
+        <version>3.1.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.confluent</groupId>
+    <artifactId>kafka-schema-registry-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+    <name>maven-plugin</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-plugin-api</artifactId>
+            <version>3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugin-tools</groupId>
+            <artifactId>maven-plugin-annotations</artifactId>
+            <version>3.2</version>
```

Comment:
Why does this version differ from the previous one? And how do we select these versions -- minimum version with the features we need so it works with as broad a range of maven versions as possible? There are newer versions of both, and matching versions at 3.0 or 3.2.